spaceship/jobs/email_job.py

```diff
@@ -0,0 +1,23 @@
+from .base_job import BaseJob
+
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+from .. import app
+
+class EmailJob(BaseJob):
+  @classmethod
+  def perform(cls, to_emails, subject, html_content, from_email='gaia@spaceshipearth.org'):
+    message = Mail(
```

Comment:
This (the Mail constructor) can throw if sendgrid key is not set up properly.

Reply:
huh i haven't seen that in my environment...